### DISCUSSION TITLE
Plugin does not work

### DISCUSSION THREAD

**Discussion by mocherry** [BrainEnergyLab/Inflammation-Index#21] [Fiji Plugin Issues]
Dear Developers,

I have installed the microglia-analysis plugin and tried to use it.
Now, what happened is, that the first time I ran Step2 (Running Stack Preprocessing) it worked. Continuing with Step3 (Running Stack QA) produced the following error: Macro Error: No window with the title "Results" found.
Then the macro is aborted.
Stranger yet: after closing and restarting everything, not even Step2 works anymore and reports the same error message.

Apart from that: Is it possible to use this pipeline for regular confocal stacks, i.e. ones which do not have multiple frames per z-slice, so motion correction would not be possible?

Help would be greatly appreciated. Thanks and best,
Matthias

**Comment by DKClarke** [comment]
Hi Matthias, apologies for the delay - could you upload an example image that is causing these errors somewhere (perhaps on Google Drive, or Dropbox) so that I can download it and replicate the problem myself?

Regarding regular confocal stacks, since motion correction is not possible what you can do instead is simply take a max or average projection of your confocal stack and run the plugin on that projection by following the instructions in the readme for single frames images (the single frame images instructions are in grey and indented) . It's not a very elegant solution, but it has worked for me in the past. Similarly, if you are happy to upload an example confocal image I can verify it works on my end.
